TRANSLATIONS.adoc

```diff
@@ -0,0 +1,13 @@
+# Translations
+
+Localisation is handled through Transifex. The project can be accessed through https://www.transifex.com/yubico-1/yubico-authenticator-android[this link].
```

Comment:
typo: Localization

Reply:
nit: I thought all process of adapting app for other countries called localization, so I would name file localization (not translation)

Reply:
Yep, better to have localization everywhere instead of translation. Localisation (with ISA) is also an accepted way of writing this word, but I think it is not common in the USA then.

Reply:
Yeah, both are "correct" but -ization is the [recommended international style](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise,_-ize_(-isation,_-ization)).